tests/unit/torch/outputs/test_tabular.py

```diff
@@ -29,6 +29,8 @@ def test_exceptions(self):
         with pytest.raises(ValueError, match="not found"):
             mm.TabularOutputBlock(self.schema, init="not_found")
 
+    def test_no_route_for_non_existent_tag(self):
         outputs = mm.TabularOutputBlock(self.schema)
-        with pytest.raises(ValueError):
-            outputs.add_route(Tags.CATEGORICAL)
+        outputs.add_route(Tags.CATEGORICAL)
```

Comment:
This was changed to not raise an error because it is possible that there is no continuous features in the DLRM block. Alternatively, we could have a try-except in the DLRM block, but this felt more natural to me.